### DISCUSSION TITLE
threejs with model viewer editor without drag

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3315] [Q&A]
First of all, I'd like to apologize. I've been thinking about it for a few days, but I couldn't get an answer, so I'm leaving an inquiry.
After implementing the glb file in threejs, I would like to load the glb file automatically without dragging it to the model viewer editor, is it possible?
For example, in the case of a shopping mall site, can you do that function with the model viewer editor, just like putting the items I want to buy in a shopping cart by clicking a button?

**Comment by elalish** [comment]
You can set the MV `src` to an ObjectURL of the the GLB you've created. Will that solve your problem?

**Reply by ghost** [reply]
sorry.. could you tell me more details?? sorry i am not good skills
